JavaErickHernandez/Day3/ArrayLists.java

```diff
@@ -0,0 +1,93 @@
+package bootcamp;
+import java.util.Iterator;
+import java.util.Scanner;
+import java.util.*;;
```

Comment:
- avoid grouping libraries
- delete that extra comma

Reply:
On my way